content/en/data_jobs/databricks.md

```diff
@@ -184,6 +184,16 @@ In Datadog, view the [Data Jobs Monitoring][6] page to see a list of all your Da
    }
    {{< /highlight >}}
 
+### Set up Data Jobs Monitoring with Databricks Networking Restrictions
+With the default [Databricks Networking Restrictions][12], Datadog may not have access to your Databricks APIs, which can prevent three key requirements for Data Jobs Monitoring from functioning properly:
+1. Ensure that the agent can be downloaded onto the Databricks cluster using the curl command in the init script.
+2. Verify that the Datadog crawler has sufficient permission to access the Databricks workspace.
+3. Confirm that traces and metrics can be sent.
+
+Allow-listing these specific [IP addresses][13] allows your cluster to perform all these interactions with Datadog services. Please see [Databricks documentation][14] for more details on how to manage IP access lists in Databricks.
+
+If you are using [Databricks Private Connectivity][15], we recommend contacting our support team.
```

Comment:
Add link to support to they can create a case easily https://www.datadoghq.com/support/